client/src/views/Login/Login.js

```diff
@@ -0,0 +1,253 @@
+    padding: '1em',
+  },
+})(Button);
+
+const LoginView = (props) => {
+  const { toggleView, ...extraProps } = props;
+  const [showPassword, setShowPassword] = React.useState(false);
+  const { register, handleSubmit, errors } = useForm({ mode: 'onChange' });
+  const handleClickShowPassword = () => {
+    setShowPassword((currentValue) => !currentValue);
+  };
+  const onSubmit = (data) => {};
+
+  return (
+    <React.Fragment>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        {...extraProps}
+        className={style.form}
+      >
+        <StyledTextField
+          name="email"
+          label="Email"
+          variant="outlined"
+          placeholder="Enter email address"
+          fullWidth
+          inputRef={register({
+            required: 'Email address is compulsory',
+            pattern: {
+              value: EmailRegex,
+              message: 'Invalid email address',
+            },
+          })}
+          error={Boolean(errors.email)}
+          helperText={errors.email && errors.email.message}
+        />
+        <StyledTextField
+          name="password"
+          label="Password"
+          variant="outlined"
+          placeholder="Enter Password here"
+          fullWidth
+          type={showPassword ? 'text' : 'password'}
+          inputRef={register({ required: 'Password field is compulsory' })}
+          error={Boolean(errors.Password)}
+          helperText={errors.Password && errors.password.message}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                <IconButton
+                  className={style.icon}
+                  aria-label="toggle password visibility"
+                  onClick={handleClickShowPassword}
+                >
+                  {showPassword ? <Visibility /> : <VisibilityOff />}
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+        />
+        <StyledButton type="submit" fullWidth>
+          <Typography variant="button">Login</Typography>
+        </StyledButton>
+      </form>
+      <HyperLink align="center" href="/forgot-password">
+        <Typography variant="subtitle1">Forgot Password</Typography>
+      </HyperLink>
+      {/*
+        GOOD-TO-HAVE : ADD Third Party Login Integration
+      <div align="center" className="thirdPartyLogin">
+        { 
+          TODO : Integrate Google Login here.
+        }
+        <StyledButton>
+          <Typography variant="button">Google</Typography>
+        </StyledButton>
+        {
+          TODO : Integrate Facebook Login here.
+        }
+        <StyledButton>
+          <Typography variant="button">Facebook</Typography>
+        </StyledButton>
+      </div>
+      */}
+      <HyperLink onClick={toggleView}>
+        <Typography variant="subtitle2" align="center">
+          New User ? Sign Up
+        </Typography>
+      </HyperLink>
+    </React.Fragment>
+  );
+};
+
+/* 
```

Comment:
remove the todos if they are completed.

Reply:
They are removed in API Integration PR. Given the short time available, it was advised by rishabh to drop these features.